packages/braid-design-system/src/lib/utils/flattenChildren.ts

```diff
@@ -0,0 +1,36 @@
+// Copied and modified from https://github.com/grrowl/react-keyed-flatten-children/blob/0b4fd6dad491905e73551c9d4496198f6b87eb41/index.ts
```

Comment:
Pulling in this function to [fix the build](https://buildkite.com/seek/metropolis/builds/13647#018623e9-978e-4f0e-bedf-4dd9b9bb73be/6-139) in consuming apps. I've copied the function and made slight tweaks to satisfy our ESLint config.

Another option would have been to move `react-keyed-flatten-children` which would have inlined the function, **as well as all of `react-is`**. `react-keyed-flatten-children` depends on `react-is@16` which is
1. a lot of dead code
2. not aligned with our version of React